File: mobileapps/android/games/HeroVSMonsterCustomView/app/src/main/java/com/petamind/herovsmonstercustomview/MainActivity.java
package com.petamind.herovsmonstercustomview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

/**
 * The simple implemetation of Custom View and touch events
 * It is NOT recommended to use this in a more complicated game.
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new GameView(this));
    }
}

File: mobileapps/android/games/Ping/app/src/main/java/com/petamind/ping/MainActivity.kt
package com.petamind.ping

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.petamind.ping.components.GameView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(GameView(this))
    }
}